modules/ebi-metagenomics/hmmer/hmmsearch/meta.yml

```diff
@@ -0,0 +1,49 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "hmmer_hmmsearch"
+description: search for sequences against a HMM profile database
```

Comment:
I think nf-core's description is a bit more accurate here, which I would change a bit in:
"Use an HMM profile to search for hits in a sequence database"